src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+const initialState = (window as any).initialReduxState;
+// const store = configureStore(initialState);
+
+function App() {
+  return (
+    <></>
+    // <Provider store={store}>
+    //   <StudentPortal />
+    // </Provider>
```

Comment:
Your structure should contain basic implementation of of redux with a component.

redux itself can be empty and component can return <></> for now.